webui/react/src/e2e/models/BasePage.ts

```diff
@@ -73,4 +73,10 @@ export abstract class BasePage implements ModelBasics {
     }
     return this;
   }
+
+  async log(s: string): Promise<void> {
```

Comment:
@djanicekpach you'll like this

Reply:
Can this be something like `pageLog` or `browserLog` so it's clear that this is logging on the browser side console?

Reply:
sure thing. i'll add a docstring too